src/components/EtaxForm/configs/AddressFields.config.js

```diff
@@ -7,46 +7,95 @@ import _ from "lodash"
  */
 const addressFields = [
   {
-    id: "houseNo",
-    label: "House No.",
-    helperText: "Your House Number",
+    id: "postCode",
+    label: "รหัสไปรษณีย์",
+    helperText: "ex. 11120, 10520",
```

Comment:
If you're going to have `ex.` in front of every helper text, how about putting them in one place so you don't have to repeat it?

Just a little thought :)